pkg/operator/encryption/migration_controller_test.go

```diff
@@ -168,8 +168,13 @@ func TestMigrationController(t *testing.T) {
 				"get:secrets:kms",
 				"list:secrets:openshift-config-managed",
 				"list:secrets:openshift-config-managed",
+				"get:secrets:openshift-config-managed",
```

Comment:
oh, wow, fewer events!

Reply:
one after each resource. So you know where it stands. Poor-men's progress view (will all be replaced with @sanchezl 's storage migration and pretty progress logging in the migration object status).